mappings/net/minecraft/block/Material.mapping

```diff
@@ -11,12 +11,12 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
-	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
-		COMMENT Not in use, but has foliage color.
+	FIELD field_15921 MOSS_BLOCK Lnet/minecraft/class_3614;
```

Comment:
Is this proper to be backported?

Reply:
I believe that yes, at the time they had to know they already wanted to add moss and maybe even had a branch with it.
The value is exactly the same (so literally no differences between 1.16 and 1.17 here), it's maybe unused in that version, but we know what it corresponds to once used, so I don't see why it should be ignored.
A javadoc explaining it's unused in that version should be enough imo.